public/Lab-Files/Lesson02/Challenges/Images.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Images</title>
+</head>
+<body>
+    <h1>The perfect trip</h1>+
+
+<p>I would go to... <img src="http://media.salon.com/2015/09/mars-614x412.jpg"></img></p>
```

Comment:
Whenever you have an image tag, you need to include the alt attribute with a text description of what the image represents. 
Also, images are void elements, so they do not need a closing tag. 

Please make sure you validate your code before committing. By doing so, you will avoid most of these errors. The validator is a great tool.